src/given/definitions.js

```diff
@@ -131,6 +133,16 @@ const stepDefinitions = [
 		regex: /^I debug$/,
 		fn: debug,
 		title: "Add a breakpoint to stop the running browser and give you time to jump into it and check the state of your application ([WDIO Help on Debug](http://webdriver.io/api/utility/debug.html))."
+	},
+	{
+		regex: /^I am logged in to (beta|live|test) Accounts with username "([A-Z0-9_]+)" and password "([A-Z0-9_]+)"$/,
```

Comment:
The `beta|live|test` bit could be optional, defaulting to live. It's probably also worth putting in an example with environment variables to make it clear how to use it

Reply:
Added an example to the title section for this step. I am not going to set the beta|live|test bit to be optional as I think we should force tester to be explicit.